lib/Interpreter/Interpreter.cpp

```diff
@@ -1177,6 +1177,27 @@ namespace cling {
   }
 
   void Interpreter::unload(Transaction& T) {
+    // Clear any stored states that reference the llvm::Module.
+    // Do it first in case
```

Comment:
"...in case" what? (Merging nonetheless, but I'd appreciate an email or a follow-up PR. I can guess but you know!)